src/main/java/frc/robot/RobotContainer.java

```diff
@@ -8,6 +8,21 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.button.*;
 import frc.robot.commands.*;
+import frc.robot.commands.ClampCmd;
+import frc.robot.commands.LowerCmd;
+import frc.robot.commands.OneBar.*;
+import frc.robot.commands.RaiseCmd;
+import frc.robot.commands.ReleaseCmd;
+import frc.robot.commands.RotLeftCmd;
+import frc.robot.commands.RotRightCmd;
+import frc.robot.commands.ShiftLeftCmd;
+import frc.robot.commands.ShiftRightCmd;
+import frc.robot.commands.StopClawCmd;
+import frc.robot.commands.StopRaiseCmd;
+import frc.robot.commands.StopRotCmd;
+import frc.robot.subsystems.*;
+import frc.robot.subsystems.ClawSubsystem;
+import frc.robot.subsystems.WristSubsystem;
```

Comment:
Same here.  Is this needed if there's a `import frc.robot.subsystems.*;` line?

Reply:
if the "import frc.robot.subsystems.*;" is there, is it importing every subsystem into the robot container